libsolidity/ast/Types.cpp

```diff
@@ -2126,7 +2126,8 @@ bool StructType::canBeUsedExternally(bool _inLibrary) const
 		// mappings.
 		for (auto const& var: m_struct.members())
 		{
-			solAssert(var->annotation().type, "");
+			if (!var->annotation().type)
+				return false;
```

Comment:
Please add a comment.

Reply:
Please also change the description of the function to add this additional "false"-case.